src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/links/BfdSpec.groovy

```diff
@@ -203,4 +202,46 @@ class BfdSpec extends HealthCheckSpecification {
             assert northbound.getLink(isl.reversed).state == IslChangeType.DISCOVERED
         }
     }
+
+    @Tidy
+    @Tags([SMOKE_SWITCHES])
+    def "System is able to rediscover failed link after deleting BFD session"() {
+        given: "An interrupted an a-switch ISL with BFD session"
```

Comment:
Didn't get this slogan. Did you mean `An interrupted a-switch ISL..`?

Reply:
my bad, fixed